python/doc/theory/reliability_sensitivity/ranking_src.rst

```diff
@@ -1,70 +1,78 @@
-and standard deviation :math:`\sigma_{\varepsilon}^j` independent of the
+where :math:`\varepsilon` describes a random variable with zero mean
+and standard deviation :math:`\sigma_{\varepsilon}` independent of the
 input variables :math:`X^i`. If the random variables
 :math:`X^1,\ldots,X^{n_X}` are independent and with finite variance
-:math:`\Var{X^k} = (\sigma_k)^2`, the variance of :math:`Y^j` can be
+:math:`\Var{X^k} = (\sigma_i)^2`, the variance of :math:`Y` can be
 estimated as follows:
 
 .. math::
 
-    \Var{Y^j} = \sum_{i=1}^n (a_i^j)^2 \Var{X^i} + (\sigma_{\varepsilon}^j)^2 = (\sigma^j)^2
+    \Var{Y} = \sum_{i=1}^n (a_i)^2 \Var{X^i} + (\sigma_{\varepsilon})^2 = (\sigma)^2.
 
-
-The estimators for the regression coefficients
-:math:`a_0^j,\ldots,a_{n_X}^j`, and the standard deviation
-:math:`\sigma^j` are obtained from a sample of
-:math:`(Y^j,X^1,\ldots,X^{n_X})`. Uncertainty ranking by linear
-regression ranks the :math:`n_X` variables :math:`X^1,\ldots, X^{n_X}`
-in terms of the estimated contribution of each :math:`X^k` to the
-variance of :math:`Y^j`:
+From this we obtain the following coefficients:
 
 .. math::
 
-    C^j_k = \frac{\displaystyle (a_k^j)^2  \Var{X^k}}{ \Var{Y^j}}
-
+    C_i = a_i \sqrt{\frac{\Var{X^k}}{\Var{Y}}}
 
-which is estimated by:
+The estimators for the regression coefficients
+:math:`a_0,\ldots,a_{n_X}`, and the standard deviation
+:math:`\sigma` are obtained from a sample of
+:math:`(Y,X^1,\ldots,X^{n_X})`.
+The SRC coefficients are defined as the estimators :math:`\widehat{C}_i`
+of the coefficients :math:`C_i`:
 
 .. math::
 
-    \widehat{C}^j_k = \frac{\displaystyle (\widehat{a}_k^j)^2 \widehat{\sigma}_k^2}{\displaystyle (\widehat{\sigma^j})^2}
+    \widehat{C}_i = \frac{\displaystyle \widehat{a}_i \widehat{\sigma}_i}{\displaystyle \widehat{\sigma}},
 
 
-where :math:`\widehat{\sigma}_i` describes the empirical standard
-deviation of the sample of the input variables. This estimated
+where :math:`\widehat{a}_i` denotes the estimate of the regression coefficient :math:`a_i`,
+:math:`\widehat{\sigma}_i` denotes the empirical standard
+deviation of the sample of the input variable :math:`X^i`
+and :math:`\widehat{\sigma}` denotes the empirical standard
+deviation of the sample of the output variable :math:`Y`.
+The absolute value of this estimated
 contribution is by definition between 0 and 1. The closer it is to 1,
 the greater the impact the variable :math:`X^i` has on the dispersion of
-:math:`Y^j`.
+:math:`Y`.
 
-The contribution to the variance :math:`C_i` is sometimes described in
+The square :math:`(C_i)^2`, which is the contribution of :math:`X^i`
+to the variance of :math:`Y`,
+is sometimes described in
 the literature as the “importance factor”, because of the similarity
 between this approach to linear regression and the method of cumulative
-variance quadratic which uses the term importance factor.
+variance which uses the term importance factor.
```

Comment:
There are two missing points here: 
- If the physical model g is linear, then squared SRC coefficients are equal to the Sobol' indices. This is what is important to see in the squared SRC coefficient. On this point, a new pointer to the [Sobol'](https://openturns.github.io/openturns/latest/theory/reliability_sensitivity/sensitivity_sobol.html) help page should be added.
- The quality of the linear regression should be checked before estimating the SRC indices. This is because if the linear regression model is a poor fit to the data, then the SRC coefficients are useless. This is why a new pointer to the [Regression](https://openturns.github.io/openturns/latest/theory/data_analysis/linear_regression.html) help page should be added, with a suggestion: compute the R2 and check it before estimating the SRC indices.

Reply:
OK, I have added this to the doc page.